File: problem-1134/1134.py
counter_alcool = 0
counter_gasolina = 0
counter_diesel = 0

while True:
    try:
        combustivel = int(input())
        if combustivel == 1:
            counter_alcool += 1
        elif combustivel == 2:
            counter_gasolina += 1
        elif combustivel == 3:
            counter_diesel += 1
        elif combustivel == 4:
            break
    except ValueError:
        pass

print("MUITO OBRIGADO")
print(f"Alcool: {counter_alcool}")
print(f"Gasolina: {counter_gasolina}")
print(f"Diesel: {counter_diesel}")

File: problems/1134.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let counterAlcohol = 0
let counterGas = 0
let counterDiesel = 0

let i = 0;

while(true){
    if (lines[i] == 1){
        counterAlcohol += 1
    } else if (lines[i] == 2){
        counterGas += 1
    } else if (lines[i] == 3){
        counterDiesel += 1
    } else if (lines[i] == 4){
        break;
    }
    i ++
}

console.log("MUITO OBRIGADO");
console.log(`Alcool: ${counterAlcohol}`);
console.log(`Gasolina: ${counterGas}`);
console.log(`Diesel: ${counterDiesel}`);
